### PR TITLE
Add more installation methods in quick start

### PR DESCRIPTION
**Pull request by xsoheilalizadeh**
These days developers haven't single option to install a nuget packages in their project therefore I've add some installation methods to installing FakeItEasy package.

### CODE REVIEW

File: docs/quickstart.md [hunk 1]
```diff
@@ -1,11 +1,34 @@
...
-Getting started with FakeItEasy is very simple:
+## Getting started with FakeItEasy
 
-* Open the Package Manager Console:  
-Tools → Library Package Manager → Package Manager Console
-* Execute `Install-Package FakeItEasy`
-* Start faking dependencies in your tests. Here's a sample test class with fakes:
+### Installation
+
+#### Package Manger
```
Comment (thomaslevesque) [inline docs/quickstart.md]: Manger  => Manager

Comment (xsoheilalizadeh) [reply docs/quickstart.md]: Fixed

**Comment by xsoheilalizadeh** [comment]
I'm agree with you somewhat, but it's almost about having a comprehensive documentation, I considered developers who are beginner and also folks who start learning FakeItEasy, they'll be comfort with copying commands and installing package.

**Comment by blairconrad** [comment]
Oh! I'd made some comments on the changes while I was still at home and hadn't read the conversation here. I didn't realize they were still active and going to be included here. You can probably ignore them.

File: docs/quickstart.md [hunk 2]
```diff
@@ -1,11 +1,34 @@
...
+```bat
+dotnet add package FakeItEasy
+```
+
+#### PackageReference
+
+ For projects that support PackageReference, copy this XML node into the csproj file to reference the package.
+
+```xml
+<PackageReference Include="FakeItEasy" Version="5.2.0" />
```
[the remark this answers is not in the document]
Comment (thomaslevesque) [reply docs/quickstart.md]: And the equivalent instructions on NuGet.org are always up to date ;)

**Comment by xsoheilalizadeh** [comment]
Thanks @blairconrad. Should I close the issue or fix change requests?

**Comment by blairconrad** [comment]
No need to apply my in-line suggested changes, @xsoheilalizadeh. Closing is fine.

However, if we've convinced you that it's a good idea to update the installation instructions by pointing at the NuGet package page, and you want to do that, we'd be thrilled to take a PR.

**Pull request closed without merging**